src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  let partnerMessage;
```

Comment:
1.   Use `const` instead of `let`
2. Use Template Literals: Use template literals for string interpolation, which can make your code more readable.
3. Use Ternary Operator for Conditional Rendering: You can simplify your conditional rendering by using a ternary operator directly in JSX.
  
 Example: 
```
  const partnerMessage = person.isMarried
    ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
    : "I am not married";
    
    <p className="Person__partner">{partnerMessage}</p>
```